tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Get Commencement Date.feature

```diff
@@ -0,0 +1,48 @@
+﻿Feature: Display the Order Commencement Date in a Buyer Section
+    As a Buyer
+    I want to view an order's commencement date
+    So that I can ensure that the information is correct
+    
+Background:
+    Given Orders exist
```

Comment:
I think Sam changed it so you dont need LastUpdated in the table, up to you if you want to remove/keep it